webpack.config.js

```diff
@@ -25,67 +24,119 @@ export default {
     hot: true
   },
   plugins: [
-    new webpack.DefinePlugin(GLOBALS),
+    new webpack.DefinePlugin(GLOBALS), //define variables to use on development
     new webpack.HotModuleReplacementPlugin(),
     new webpack.NoEmitOnErrorsPlugin(),
     new webpack.LoaderOptionsPlugin({
       debug: true
     })
   ],
   module: {
-    rules: [
-      {
-        test: /\.js$/,
-        include: path.join(__dirname, 'app'),
-        use: [{
-          loader: 'babel-loader'
-        }]
-      },
-      {
-        test: /(\.css)$/,
-        use: [{
-          loader: 'style-loader'
-        },
-        {
-          loader: 'css-loader'
-        }]
-      },
-      {
-        test: /\.eot(\?v=\d+\.\d+\.\d+)?$/,
-        use: [{
-          loader: 'file-loader'
-        }]
-      },
-      {
-        test: /\.(woff|woff2)$/,
-        use: [{
-          loader: 'url-loader',
-          options: {
-            prefix: 'font',
-            limit: 5000
-          }
-        }]
-      },
-      {
-        test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/,
-        use: [{
-          loader: 'url-loader',
-          options: {
-            limit: 10000,
-            mimetype: 'application/octet-stream'
+    rules: [{
+      test: /\.js$/,
+      include: path.join(__dirname, 'app'),
+      use: [{
+        loader: 'babel-loader'
+      }]
+    }, {
+      enforce: 'pre',
+      test: /\.scss/,
+      loader: 'import-glob-loader'
+    }, {
+      test: /(\.css)$/,
+      use: [{
+        loader: 'style-loader'
+      }, {
+        loader: 'css-loader'
+      }]
+    }, {
+      test: /(\.scss)$/,
+      use: [{
+        loader: 'style-loader'
+      }, {
+        loader: 'css-loader'
+      }, {
+        loader: 'postcss-loader',
```

Comment:
Asking out of ignorance: is `css-loader` required if we use `postcss-loader`?

Reply:
Not sure here @AlphaGit. This was a mix of magic with @arzafran when we created the webpack config. The same that we are using in VAS. Not wanting to try different things in this PR. Only the config that is already tested and working ok. Thanks

Reply:
Sounds good -- thanks!

I'm fine with playing it safe, and if we think about it in an iterative way, then definitely, let's go ahead. However, I'd like to point out that we do expect contributors to have a good understanding of the changes and their impact. We should be able to understand benefits and disadvantages so that we can make the correct choices.

As I said: I'm fine with it, but I needed to point out that this needs to be the exception rather than the rule.

Also, I hope you don't mind, but I redacted your comment to remove our customer's name. We mentioned this in another and I saw your agreement so I think you are fine with it.